app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -781,6 +781,9 @@ export default {
     },
     setReplyMode(mode = REPLY_EDITOR_MODES.REPLY) {
       const { can_reply: canReply } = this.currentChat;
+      this.$store.dispatch('draftMessages/setReplyMEditorMode', {
```

Comment:
is the `M` in setReply**M**EditorMode intentional, feels like a typo

Reply:
Yes. Fixed.